torch/csrc/jit/passes/utils/subgraph_utils.cpp

```diff
@@ -202,20 +212,30 @@ void mergeNodeIntoSubgraph(
 
   auto subgraph = getSubgraph(subgraphNode);
 
-  // Map from values in the surrounding graph to inputs in the subgraph
-  std::unordered_map<Value*, Value*> inputsMap;
+  // Map from values in the surrounding graph to values in the subgraph
```

Comment:
Minor: tbh, this comment is hard to follow. Maybe add an example or change "values" to "inputs/outputs".